tests/testAutopas/tests/containers/AllContainersTests.cpp

```diff
@@ -64,8 +64,9 @@ TEST_P(AllContainersTests, testParticleAdding) {
                        boxMax[2] + .5, boxMax[2] + 1.5}) {
         autopas::Particle p({x, y, z}, {0., 0., 0.}, id++);
         if (x == -1.5 or y == -1.5 or z == -1.5 or x == 11.5 or y == 11.5 or z == 11.5) {
-          EXPECT_ANY_THROW(container->addParticle(p));     // outside, therefore not ok!
-          EXPECT_NO_THROW(container->addHaloParticle(p));  // much outside, still ok because it is ignored!
+          EXPECT_ANY_THROW(container->addParticle(p));  // outside, therefore not ok!
+          EXPECT_NO_THROW(container->addHaloParticle(p))
+              << " x=" << x << " y=" << y << " z=" << z;  // much outside, still ok because it is ignored!
```

Comment:
you can also directly print the whole particle. I think like this:
```suggestion
              << p.tostring();  // much outside, still ok because it is ignored!
```

Reply:
Since this was only used for debugging purposes, I reverted it back to its original state in da1abe0.